docs/reference/method/MongoDBGridFSBucket-openUploadStream.txt

```diff
@@ -40,7 +40,25 @@ Behavior
 Chunk documents will be created as data is written to the writable stream. The
 metadata document will be created when the writable stream is closed.
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   $stream = $bucket->openUploadStream('filename');
```

Comment:
I might suggest differentiating these as `$uploadStream` and `$downloadStream` below.